### PR TITLE
Mihai/Homepage

### PR DESCRIPTION
**Pull request by iftodimihail**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/107#" title="Author self-approved">iftodimihail</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/website/gatsby-node.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ exports.createPages = ({ graphql, actions }) => {
...
-    toPath: `/introduction`,
-    redirectInBrowser: true,
-    isPermanent: true,
-  })
+  // createRedirect({
+  //   fromPath: `/`,
+  //   toPath: `/introduction`,
+  //   redirectInBrowser: true,
+  //   isPermanent: true,
+  // })
```
Comment (codrin-iftimie) [inline packages/website/gatsby-node.js:16]: remove the lines

**Comment by codrin-iftimie** [comment]
This requires some work. Currently, the latest updates query is mocked.
We need a way to inject data into mdx components
Gatsby only lets up use pages or templates to do queries. Mdx components are disconnected from the page.
Looking into solutions for this

**Comment by saamalik** [comment]
@iftodimihail how's this lookIng?

**Comment by codrin-iftimie** [comment]
@saamalik All good. We only need to update the content.

**Comment by codrin-iftimie** [comment]
Closing this in favor of https://github.com/spectrocloud/librarium/pull/124

**Pull request closed without merging**